### DISCUSSION TITLE
CoD 4 Weapons?

### DISCUSSION THREAD

**Discussion by Revnova** [nzp-team/nzportable#405] [General]
i'm writing this kind of as a suggestion to anyone who can do this. i just think this could be a cool nzp mod. cod4 weapons. with scopes and other attachments. even if its just models with the animations that replace current guns instead of the whole guns being written would be cool. and i mean the original cod4. not the remake or mw19. guns i really like are the M4, G36C, MP5, Skorpion, P90, M249 Saw, RPD, and M60. so any of these would be great at all.

**Comment by pahntomforces-guy** [comment]
hell yes
